### ISSUE TITLE
Style and Design

### ISSUE THREAD

**Issue by austintgriffith** [austintgriffith/burner-wallet#5]
The current design/UI is very ugly. It was a quick proof-of-concept. Please create and implement a gorgeous and intuitive design. 

We want it to look great and work intuitively even in different languages. 

This is a big task and will take a lot of thought and style.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 100.0 DAI (100.0 USD @ $1.0/DAI)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $56,649.44 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by shailysangwan** [comment]
Hi, I have applied for this issue on Gitcoin but it seems like the corresponding comment from @gitcoinbot hasn't showed up here yet.

**Comment by shailysangwan** [comment]
Hi, I have applied for this issue on Gitcoin but it seems like the corresponding comment from @gitcoinbot hasn't showed up here yet.

I'm planning on these lines - 
Since the application is built in react, we can consider using a component library like Material UI or Blueprint JS.
I'll prepare a mockup in one of those, discuss improvements with @austintgriffith and complete the design accordingly.

**Comment by joshpitzalis** [comment]
@shailysangwan if you need any help, let me know. I was considering taking this on but it looks like you're on it. 

There is no need to use a component library. There are so few components you can write them custom. 

I found 2 nice references that I thought would be good for the project 
https://dribbble.com/shots/4134552-Masternode-Crypto-Coin-Wallet-Redesign
https://dribbble.com/shots/4134609-Phore-Crypto-Coin-Wallet-Preview

...and potentially this logo to replace the emoji https://thenounproject.com/term/flame/344414/

Just thoughts. 

Stuff I found when I was considering taking it on.

Also, I think this is a great project to use vh units. So that you can get the whole app in a single screen, no scrolling. The fun bit would be adapting the screen to mobile so there is also no scroll on mobile.

Anyway best of luck with it. 

I had a look at the source  and its looks like you have quite the refactor ahead of you.

**Comment by austintgriffith** [comment]
Thanks for the interest guys! I would love to see some mockups for how we can make this app look stunning. Please hit me up here, or on twitter/telegram @austingriffith with any questions. 

If we have two mockups going in parallel I would be happy to reward that optionality with $25 DAI to the one we don't select just for a thank you for your efforts. It's important that this design be incredibly intuitive even for non-english speakers. 

Also, yes, the code is all in one big App.js file right now because I wrote most of it in the airport on the way to devcon :) There is another bounty to refactor it and I'm confident it will be solid soon.

With that in mind, ideally, the designer who creates the mockup can also implement it into the React code. Thanks guys! Reach out to me on twitter/telegram and let's talk more!

**Comment by willsputra** [comment]
@austintgriffith I love the concept! Excited to see this being worked on ❤️

Just wanted to throw a couple of possible user questions worth thinking about regarding UX:
1. Can I only send DAI to the address? Can I send ETH? What happens if I send ETH?
2. Since 'Burn Private Key' is such a 'destructive' button, we might want to put a bit more description for the users on what the button actually does.
It might also be a good idea to add a second confirmation step ('Are you sure you want to burn your private key? Your wallet will be gone blah blah' [Yes] [Cancel]) to prevent users from accidentally tapping on the button.

All the best to whoever's doing the design 🙂

**Comment by joshpitzalis** [comment]
Didn't realise design and implementation were two separate bounties. The bounty description made it sound like the same gig. The comment above makes it sound like its two different jobs. I'ma  little confused. Either way here is a mock up of what I was thinking.

![xdai](https://user-images.githubusercontent.com/10258114/48280431-91050080-e479-11e8-951b-a77e45872a14.png)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 1 week, 6 days from now.
Please review their action plans below:


**1) [shailysangwan](https://gitcoin.co/profile/shailysangwan) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715?mutate_worker_action=approve&worker=shailysangwan) | [reject worker](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715?mutate_worker_action=reject&worker=shailysangwan))_.**

Since the application is built in react, we can use a component library like Material UI or Blueprint JS.
I'll create a mockup using one of these, discuss improvements with the mentor and complete the design accordingly.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).


**2) [joshpitzalis](https://gitcoin.co/profile/joshpitzalis) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715?mutate_worker_action=approve&worker=joshpitzalis) | [reject worker](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715?mutate_worker_action=reject&worker=joshpitzalis))_.**

Is this a design only issue? I don't understand if implementation is funded separately. Please clarify.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).

**Comment by figure8labs** [comment]
I'd like to throw my hat in the ring if I can be of use! I've been looking for a place to get started on this project.  If it's getting too crowded of course I'll find something else.  I also don't mind just brainstorming on design ideas.  I'll post something later : )

**Comment by joshpitzalis** [comment]
@austintgriffith I'm guessing zero feedback on my initial sketch means you don't like it. Do I still get the 25 DAI for not being selected like you said?

**Comment by austintgriffith** [comment]
Oh no! Sorry! Everything is all good. I've been staying away from Git for the weekend :)
I like the design but I'd like to see more too. I'm happy to send 25 USD for your efforts! 
One thing to keep in mind is this is a mobile first app.
We should design it to look best on a phone and then expand it out to Desktop use.

**Comment by joshpitzalis** [comment]
Thanks @austintgriffith my address is 0xD9b74f73d933Fde459766f74400971B29B90c9d2

Here is a mobile example of the same:

![mobile](https://user-images.githubusercontent.com/10258114/48329051-e9e8ba80-e66c-11e8-90a4-5c01ff3acf46.png)

The plan is to adjust the QR code so that that screen always equals the height of the screen.

The UI is a little longer because it includes the alert, which would temporarily push the screen down while active.

I'll stay in the loop incase you want me to build this out. If I do end up writing this out, I'll have to take the refactoring gig as well because it would be super hard to do one without the other.

You have a hoard of if/else statements. State machine are particularly good for situations like this. 

If i do end up working on this, how do you feel about using a state machine to handle react state for the refactor? I'd probably use xstate https://github.com/davidkpiano/xstate or react-automata https://github.com/MicheleBertoli/react-automata#readme. I don't have to, it's just a really good fit for this scenario. 

Have you ever worked with state charts or would you prefer I don't add the extra library?

**Comment by austintgriffith** [comment]
Sorry for the long delay over the weekend. I've approved multiple designers and I'd like to get multiple designs pitched to this thread. There are new funders joining the project and it is very important that the product be both beautiful and intuitive. Having the optionality behind different designs will enable this. We will try to reward everyone accordingly for their work!

**Comment by gitcoinbot** [comment]
💰 A crowdfund contribution worth 50.00000  DAI (50.0 USD @ $1.0/DAI) has been attached to this funded issue  from @mikemcdonald.💰 

Want to chip in also? Add your own contribution [here](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).

**Comment by gitcoinbot** [comment]
💰 A crowdfund contribution worth 100.00000  DAI (100.0 USD @ $1.0/DAI) has been attached to this funded issue  from @mkosowsk.💰 

Want to chip in also? Add your own contribution [here](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).

**Comment by mkosowsk** [comment]
100 DAI from the ECF Web 3.0 Infrastructure Fund! Looking forward to the results :)

**Comment by austintgriffith** [comment]
Lot's of funds for this issue! I'm very excited to see designs! Feel free to post early mockups here in github to gauge community reaction!

**Comment by starkblaze01** [comment]
@gitcoinbot, working on it.

**Comment by mattybgits** [comment]
Unfortunately I don't have a lot of coding experience but wanted to throw a design together. Most flows will be the same as the existing application.

- Minimal text so it can easily be translated
- Super simple

One thing we might want is a small explanation that can be easily translated for a couple of key things: 

- The gist of what it is
- What burning does
- The xDia bridge

![MobileWebxDai](https://i.imgur.com/bSqnX9F.png)

Don't seem to be able to 'Start Work' on GitCoin which is slightly odd.

**Comment by austintgriffith** [comment]
Just updated the bounty so you can start work :)

**Comment by figure8labs** [comment]
Whoa there's a lot of activity here, cool.  Sorry for the delay, this week was overwhelming, going to post later today!

**Comment by austintgriffith** [comment]
Yes! Please post designs here! Even if you are unable to implement them into React we'd love to see your work. Perhaps we can split this into two bounties; one for design and one for implementation. If you are interested in getting started with implementation, checkout the starter guide I tweeted about:  https://twitter.com/austingriffith/status/1062488114948595713

**Comment by mattybgits** [comment]
Tidied it up. No idea what posting etiquette is on Github, tell me to cool my jets if I'm spamming. Starting on interactions now, aiming to have something up on InVision by Friday.

![V2](https://i.imgur.com/qZTcjPb.png)

**Comment by seichris** [comment]
Made a design, specifically to make it easy to use the wallet for a new user, without reading medium posts or watching videos- They are great, but I think a product should aim to explain itself.

The design guides the user through the [whole life circle](https://github.com/austintgriffith/burner-wallet/issues/6) of a xDai wallet.

Also I am giving the user the choice to switch between _sender_ and _receiver_, since that saves space on the screen. (And explains what the app can do, on first view.)

### Sender

![send](https://user-images.githubusercontent.com/25006584/48552916-60d6bb00-e8da-11e8-9f35-f0bec3d1d156.jpg)


### Receiver

![receive](https://user-images.githubusercontent.com/25006584/48552921-63d1ab80-e8da-11e8-94b7-c558f7191829.JPG)


Notes on implementation:
- The site saves the users position. So a sender who got xDAI in his wallet does not have to scroll to his wallet.
- The user sees the active area, while the next area is ~opaque. Like this:

![send scroll example](https://user-images.githubusercontent.com/25006584/48553085-d773b880-e8da-11e8-89b5-b7c423f06a77.JPG)


_PS: Let me know what you think. I don't plan to implement, but you can built on top of my [figma file](https://www.figma.com/file/WpGZCNI6QWzdswT4KTohfqJt/Burner-Wallet?node-id=0%3A1)._

**Comment by joshpitzalis** [comment]
@seichris This is cool. 
I thought about adding new functionality to the app in my designs but then I wasn't sure if that was part of the brief. I wanted to add a tx history to my designs, I thought that would be useful. Persisting the data would be tricky, we have to add a db to the mix and then decide on an authentication strategy. We could just do local storage to try the feature out first though. I decided to only design for features that were already coded. 
@austintgriffith whats the brief here? If we can just draw in new functionality that opens things up quite a bit.

**Comment by seichris** [comment]
Also thought about keeping the code changes minimal - so i only added the switch on top and the feature to save the position.

You are right though. Maybe we can design a minimal one for the gitcoin issue and discuss advanced ones for advanced uses.

**Comment by jakerockland** [comment]
@seichris This design is really nice, very rad. Any way to export from Figma -> Sketch?

**Comment by seichris** [comment]
@jakerockland Thanks! Figma -> Sketch is not possible. Only svg export. You could copy-pasta my figma into a new figma file though.

**Comment by joshpitzalis** [comment]
@austintgriffith is there a deadline for when you will want to start the design work? I don't have time to work on this any more. I'll be available again on the 14th of December. Any chance you could put a rough deadline down and then we can schedule around it?

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 100.0 DAI (100.0 USD @ $1.0/DAI) has been submitted by__:

 1. [@seichris](https://gitcoin.co/profile/seichris)



@austintgriffith please take a look at the submitted work:
* [PR](https://github.com/austintgriffith/burner-wallet/issues/5#issuecomment-439024837) by @seichris

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $43,834.81 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 100.0 DAI (100.0 USD @ $1.0/DAI) has been submitted by__:

 1. [@seichris](https://gitcoin.co/profile/seichris)



@austintgriffith please take a look at the submitted work:
* [PR](https://github.com/austintgriffith/burner-wallet/issues/5#issuecomment-439024837) by @seichris

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $43,934.81 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by seichris** [comment]
Oops. Shot too early. Didn't implement anything yet.

**Comment by joshpitzalis** [comment]
We're supposed to start work? I thought one of the design is going to be picked first?

**Comment by seichris** [comment]
Ah k. Then a deadline definitely would make sense.

**Comment by austintgriffith** [comment]
Yeah let's get a couple good designs and let the community/collaborators/contributors decide what works best. Then, phase two will start and we can implement the design in React.

Designers will be paid a portion of the bounty once we select a good design and the implementor will be paid the remaining amount to get it in React.

**Comment by austintgriffith** [comment]
I think @mattybgits' design is my favorite so far but I would love to see some feedback from the community. Should we reward him and continue on to implementing? Anyone out there want to start the second half of this bounty and start implementing the design in React?

**Comment by mattybgits** [comment]
Apologies for dropping off the grid. Just moved to Berlin. If everyone
likes it, happy to finish up the screens and get them uploaded tomorrow!

**Comment by shailysangwan** [comment]
@austingriffith I'd like to do the second half!

**Comment by mattybgits** [comment]
@austintgriffith + everyone else. InVision prototype for the 3 sending processes. Hotspots should be working. Working out copy for saving and deleting, a little info modal + an explanation of the xDai Bridge. Let me know what you think, would appreciate some feedback. 

https://invis.io/NFP3H7LUK2T

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 50.00000  DAI (50.0 USD @ $1.0/DAI) has been granted to @mattybgits for this issue from @austintgriffith. ⚡️ 

Nice work @mattybgits! Your tip has automatically been deposited in the ETH address we have on file.

 * $33517.91 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__This Bounty has been completed.__

Additional Tips for this Bounty:
* austintgriffith tipped 50.0000 DAI worth 50.0 USD to mattybgits.

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $33,417.91 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__This Bounty has been completed.__

Additional Tips for this Bounty:
* austintgriffith tipped 50.0000 DAI worth 50.0 USD to mattybgits.

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/austintgriffith/burner-wallet/5/1715)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $33,417.91 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by austintgriffith** [comment]
This issue is back open again. Sorry, I don't know how to use Gitcoin. I'd love to see the second half of this bounty! I think @shailysangwan might be working on getting the design implemented in React?

**Comment by shailysangwan** [comment]
Yup! Should be complete by Thursday.

**Comment by mattybgits** [comment]
@shailysangwan If you need anything, let me know! Can send through style info and should have the rest of the screens uploaded to InVision by tomorrow early afternoon at the latest.

**Comment by davecraige** [comment]
👋 Hi everyone, does anyone want to start a Figma file for all these great ideas and designs?  Happy to jump in and help.

**Comment by mattybgits** [comment]
@shailysangwan @davecraige I've imported my Sketch document to Figma. 

https://www.figma.com/file/foeT5XpJAbkOutTRHPQWga2Z/Dai?node-id=0%3A127

Font requirements:

"FontAwesome5FreeSolid", "ArialRoundedMTBold", "InterUI-Bold", "InterUI-Regular", "InterUI-Medium", "InterUI-ExtraBold", "InterUI-SemiBold"

All the icons are done through FontAwesome so should be very easy for development.

**Comment by joshpitzalis** [comment]
Hey there, just wondering what the status is on this project.
@mattybgits congratulations on winning the gig. Great work. Are you also working on desktop and tablet designs or just the mobile views?
@shailysangwan are you still working on dev? whats the ETA? Do you need any help? Here is you need me. Happy to work on this if you decide not to.
@austintgriffith I sent you my ethereum wallet id but I never got paid for runner up design work? Do I need to rejoin the gitcoin gig? I'm unclear how this works. Please let me know.

**Comment by shailysangwan** [comment]
@joshpitzalis I've been working on it, it'll be completed soon.

**Comment by davecraige** [comment]
@mattybgits nice work on the designs!  Great hustle.  (Any thoughts on also making the file editable to others?)

Also what is your Twitter account again?  Mine is http://www.twitter.com/DaveCraige

**Comment by starkblaze01** [comment]
Is somebody still working on the implementation part?
I can help with on it.

**Comment by mattybgits** [comment]
@davecraige Sorry! Didn't realise that the Figma document wasn't editable. I'll enable that!

Twitter is https://twitter.com/mattybchats

@starkblaze01 not entirely sure what's going on implementation wise. I tried to get in touch with @shailysangwan the other day, but no response so far.

**Comment by shailysangwan** [comment]
It's done for the most part, I'll give it another 3-4 hours to take one final shot at it and make a PR for whatever I've done.

**Comment by austintgriffith** [comment]
Just merged in the redesign from @shailysangwan. There are still some things TODO but I love how it looks so far!

**Comment by gitcoinbot** [comment]
<table>
<tr>
<td>
<a href='https://gitcoin.co/kudos/385/creative_cat'><img src='https://gitcoin.co/dynamic/kudos/385/creative_cat'></a>
</td>
<td>
⚡️ A *Creative Cat* Kudos has been sent to @shailysangwan for this issue from @austintgriffith. ⚡️ 

Nice work @shailysangwan! 
Your Kudos has automatically been sent in the ETH address we have on file.
</td>
</tr>
</table>

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 50.00000  DAI (50.0 USD @ $1.0/DAI) has been granted to @shailysangwan for this issue from @austintgriffith. ⚡️ 

Nice work @shailysangwan! Your tip has automatically been deposited in the ETH address we have on file.

 * $43787.23 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by joshpitzalis** [comment]
@austintgriffith the people who did design work, but didn't get picked. You said you were going to pay us $25 for our efforts. I've checked in about this twice now and I never get a response.  Is that still on the table?

**Comment by austintgriffith** [comment]
Yes of course, sorry for the slow responses.

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 25.00000  DAI (25.0 USD @ $1.0/DAI) has been granted to @seichris for this issue from @austintgriffith. ⚡️ 

Nice work @seichris! Your tip has automatically been deposited in the ETH address we have on file.

 * $45287.07 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 25.00000  DAI (25.0 USD @ $1.0/DAI) has been granted to @joshpitzalis for this issue from @austintgriffith. ⚡️ 

Nice work @joshpitzalis! To redeem your tip, login to Gitcoin at https://gitcoin.co/explorer and select 'Claim Tip' from dropdown menu in the top right, or check your email for a link to the tip redemption page. 

 * $45287.07 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 200.00000  DAI (200.0 USD @ $1.0/DAI) has been granted to @shailysangwan for this issue from @gitcoinbot. ⚡️ 

Nice work @shailysangwan! Your tip has automatically been deposited in the ETH address we have on file.

 * $50304.88 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 109.00000  DAI (109.0 USD @ $1.0/DAI) has been granted to @mattybgits for this issue from @gitcoinbot. ⚡️ 

Nice work @mattybgits! Your tip has automatically been deposited in the ETH address we have on file.

 * $50304.88 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
